### PR TITLE
Add github action: build

### PR DESCRIPTION
**Pull request by yanoom**
Add badges to README.md: build and codecod

### CODE REVIEW

**Comment by yanoom** [comment]
See it "live" on:  
https://github.com/yanoom/pyctuator
(actions and README.md badges)

File: .github/workflows/python_package_build.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      - run: poetry install --extras flask --extras fastapi --extras db --extras redis
+      - run: make coverage
+      - uses: actions/upload-artifact@v2
+        with:
+          name: htmlcov.zip
+          path: htmlcov/
+      # - run: poetry run pytest --cov=./ --cov-report xml:./cov.xml
+      - run: ls -la
+      - uses: codecov/codecov-action@v1
+        #with:
```
Comment (MatanRubin) [inline .github/workflows/python_package_build.yml:31]: Please remove commented code

File: .github/workflows/python_package_build.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      - run: poetry build -vvv
+      - run: make package
+      - run: poetry install --extras flask --extras fastapi --extras db --extras redis
+      - run: make coverage
+      - uses: actions/upload-artifact@v2
+        with:
+          name: htmlcov.zip
+          path: htmlcov/
+      # - run: poetry run pytest --cov=./ --cov-report xml:./cov.xml
+      - run: ls -la
```
Comment (MatanRubin) [inline .github/workflows/python_package_build.yml:29]: redundant

File: .github/workflows/python_package_build.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+          #flags: unittests # optional
+          #name: codecov-umbrella # optional
+          #fail_ci_if_error: true # optional (default = false)      - uses: actions/upload-artifact@v2
+
+      # Install the extra psutil module and run the tests in "with psutil" env
+      - run: poetry install --extras psutil
+      - run: make test
+
+      # Perform linting after all dependencies have been installed!
+      - run: make check
```
Comment (MatanRubin) [inline .github/workflows/python_package_build.yml:43]: `make check` usually run quicker than the tests, so I'd put it before `make test`

**Comment by MatanRubin** [comment]
@yanoom , please improve the commit message.
Also, your commit should get rid of the gitlab-ci file, no?

**Comment by yanoom** [comment]
Made the commented adjustments.
Issued a new PR from a branch to free the master-branch:
#3

**Pull request closed without merging**